inc/Engine/Optimization/DynamicLists/AbstractDataManager.php

```diff
@@ -20,13 +29,27 @@ public function __construct( $filesystem = null ) {
 		$this->filesystem = is_null( $filesystem ) ? rocket_direct_filesystem() : $filesystem;
 	}
 
+	/**
+	 * Get cache transient name.
+	 *
+	 * @return string
+	 */
+	abstract protected function get_cache_transient_name();
+
+	/**
+	 * Get lists json filename.
+	 *
+	 * @return string
+	 */
+	abstract protected function get_json_filename();
+
 	/**
 	 * Gets the lists content
 	 *
-	 * @return object
+	 * @return object|string
```

Comment:
because here:
https://github.com/wp-media/wp-rocket/blob/dbebfe38485d7dda81555aeccda08fffda2d2bbb/inc/Engine/Optimization/DynamicLists/AbstractDataManager.php#L63

we return empty string.
What do u suggest?

Reply:
We can return a stdClass object instead of an empty string